### ISSUE TITLE
Filelist - Fix triple scrollbars on mobile

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#40945]
on small displays and If in grid view and opening the dropdown on the right it shows multiple scroll bars:
![image](https://github.com/nextcloud/server/assets/42591237/e1874d44-da06-4901-8964-19da271e59dd)

_Originally posted by @szaimen in https://github.com/nextcloud/server/pull/40917#pullrequestreview-1681992556_

**Comment by Ornanovitch** [comment]
I could be able to tackle this one but I cannot reproduce on 29.0.6. Can anyone still reproduce this bug?

**Comment by joshtrichards** [comment]
I don't believe this is an issue anymore. Or I'm unable to reproduce it in any case. @szaimen You still able to reproduce this?

**Comment by szaimen** [comment]
Yes seems to be fixed at least on 31.0.1 :)
